ccloud/beginner-cloud/start.sh

```diff
@@ -109,16 +109,13 @@ if [[ $status != 0 ]]; then
 fi
 
 echo -e "\n# Produce 10 messages to topic $TOPIC1"
-echo '(for i in `seq 1 10`; do echo "${i}" ; done) | \'
-echo "timeout 10s ccloud kafka topic produce $TOPIC1"
-(for i in `seq 1 10`; do echo "${i}" ; done) | timeout 10s ccloud kafka topic produce $TOPIC1
```

Comment:
does the removal of the usage of `timeout` have a side effect that we need to consider?

Reply:
with the timeout in place the producer might exit before all messages are sent

Reply:
Nice observation.  I'm wondering about the failure case, does the producer have it's own built in timeout to handle misconfigurations or timeouts for us so we don't just hang here indefinitely?